app/models/schedule.rb

```diff
@@ -68,20 +68,22 @@ def valid_room
   end
 
   def valid_time
-    if !start_time.blank? && !finish_time.blank? && start_time >= finish_time
-      errors.add :start_time, I18n.t("invalid_start_time")
-    end
+    unless start_time.blank? || finish_time.blank?
```

Comment:
validates :start_time, presence: true
validates :finish_time, presence: true
Will take care of it